torch/_subclasses/fake_tensor.py

```diff
@@ -759,6 +776,15 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
                 if r is not NotImplemented:
                     return r
 
+        # invoke aten-aten decomps
+        if (
+            func in decomposition_table
+            and aten_to_aten_decomp(func)
+            and func not in _disabled_meta_decomps
```

Comment:
@Chillee @SherlockNoMad not sure exactly how this plays in the new one global registration table world order